diff -ur console-tools-0.3.3/contrib/codepage.c console-tools-0.3.3-multiline/contrib/codepage.c
--- console-tools-0.3.3/contrib/codepage.c	1999-08-25 23:27:40.000000000 +0200
+++ console-tools-0.3.3-multiline/contrib/codepage.c	2003-10-23 21:58:26.783876888 +0200
@@ -228,9 +228,8 @@
 	if (optc)
 	  return 0;
 
-        fprintf(stderr, "\
-Warning: CP format is a hack!\n
-         The files produced may or may not be usable!\n");
+        fprintf(stderr, "Warning: CP format is a hack!\n"
+         "The files produced may or may not be usable!\n");
 
 	sprintf(outfile, "%d.cp", CPEntryHeader.codepage);
 	if ((out = fopen(outfile, "w")) == NULL) {
